prog/dftb+/prg_dftb/main.F90

```diff
@@ -915,6 +1045,30 @@ subroutine handleCoordinateChange(env, coord0, latVec, invLatVec, species0, mCut
   end subroutine handleCoordinateChange
 
 
+
+#:if WITH_TRANSPORT
+  subroutine initNegfStuff(negfStr, transpar, ginfo, neighborList, nNeighbor, img2CentCell, &
+              & iAtomStart, orb)
+    type(TNegfStructure), intent(in) :: negfStr
+    type(TTransPar), intent(in) :: transpar
+    type(TNEGFInfo), intent(in) :: ginfo
+    type(TOrbitals), intent(in) :: orb
+    integer, intent(in) :: img2CentCell(:)
+    integer, intent(in) :: iAtomStart(:)
+    type(TNeighborList), intent(in) :: neighborList
+    integer, intent(in) :: nNeighbor(:)
+ 
+    ! known issue about the PLs: We need an authomatic partitioning
+    call negf_init_csr(iAtomStart, neighborList%iNeighbor, nNeighbor, img2CentCell, orb)
+        
+    call negf_init_str(negfStr, transpar, ginfo%greendens, neighborList%iNeighbor, nNeighbor, img2CentCell)
+        
+    call negf_init_dephasing(ginfo%tundos)  !? why tundos 
```

Comment:
Assume this comment is asking if there is any reason not to pass in the whole structure?